src/twisted/newsfragments/11806.bugfix

```diff
@@ -0,0 +1,2 @@
+.pre-commit-config.yaml now references isort revision 5.12.0
```

Comment:
This should be a `.misc` file.

The .bugfix is only for fixes to the twisted changes that affect end users.

this is a CI / dev only change.

No need to spam Twisted users with internal changes.
We have the git history for that :)